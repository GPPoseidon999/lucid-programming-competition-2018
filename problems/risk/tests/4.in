100
400 500
401 500
402 500
403 500
404 500
405 500
406 500
407 500
408 500
409 500
410 500
411 500
412 500
413 500
414 500
415 500
416 500
417 500
418 500
419 500
420 500
421 500
422 500
423 500
424 500
425 500
426 500
427 500
428 500
429 500
430 500
431 500
432 500
433 500
434 500
435 500
436 500
437 500
438 500
439 500
440 500
441 500
442 500
443 500
444 500
445 500
446 500
447 500
448 500
449 500
450 500
451 500
452 500
453 500
454 500
455 500
456 500
457 500
458 500
459 500
460 500
461 500
462 500
463 500
464 500
465 500
466 500
467 500
468 500
469 500
470 500
471 500
472 500
473 500
474 500
475 500
476 500
477 500
478 500
479 500
480 500
481 500
482 500
483 500
484 500
485 500
486 500
487 500
488 500
489 500
490 500
491 500
492 500
493 500
494 500
495 500
496 500
497 500
498 500
499 500