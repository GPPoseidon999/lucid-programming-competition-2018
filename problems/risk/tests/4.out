0.185782
0.194129
0.203378
0.212166
0.221885
0.231099
0.241270
0.250891
0.261490
0.271494
0.282495
0.292855
0.304225
0.314911
0.326616
0.337592
0.349595
0.360824
0.373081
0.384525
0.396993
0.408608
0.421241
0.432984
0.445733
0.457560
0.470378
0.482243
0.495078
0.506936
0.519741
0.531546
0.544273
0.555980
0.568581
0.580149
0.592578
0.603965
0.616179
0.627346
0.639305
0.650217
0.661882
0.672505
0.683843
0.694147
0.705127
0.715086
0.725680
0.735271
0.745457
0.754661
0.764420
0.773221
0.782537
0.790923
0.799786
0.807748
0.816151
0.823685
0.831623
0.838727
0.846201
0.852877
0.859889
0.866142
0.872699
0.878535
0.884646
0.890075
0.895751
0.900784
0.906039
0.910690
0.915538
0.919821
0.924280
0.928212
0.932299
0.935897
0.939631
0.942913
0.946313
0.949297
0.952384
0.955087
0.957881
0.960323
0.962843
0.965042
0.967308
0.969282
0.971313
0.973080
0.974895
0.976471
0.978088
0.979489
0.980925
0.982168
